styles/components/Internship/InternshipTimeline.scss

```diff
@@ -0,0 +1,77 @@
+.timeline {
+  background-color: #CCC;
+  height: 4px;
+  margin: 50px 60px;
+  margin-bottom: 100px;
+}
+  
+.timeline-progress {
+  background-color: $acm-cobalt;
+  height: 100%;
+  width: 0;
+}
+  
+.timeline-items {
+  display: flex;
+  justify-content: space-between;
+  margin-left: -10px;
+  margin-right: -10px;
+  margin-top: -12px;
+}
+  
+.timeline-item {
+  position: relative;
+}
+  
+.timeline-item::before {
+  background-color: #CCC;
+  border-radius: 100%;
+  content: '';
```

Comment:
Could we leave some comments on what this code is doing? Stuff with pseudoselectors can definitely be confusing!